PWGGA/GammaConvBase/AliConversionPhotonCuts.cxx

```diff
@@ -2858,6 +2858,48 @@ Bool_t AliConversionPhotonCuts::SetRCut(Int_t RCut){
     fExcludeMinR = 55.;
     fExcludeMaxR = 72.;
     break;
+  case 23: //n
+    fMaxR = 180.;
+    fMinR = 10.;
+    fExcludeMinR = 180.;
+    fExcludeMaxR = 250.;
+    break;
+  case 24: //o
+    fMaxR = 180.;
+    fMinR = 15.;
+    fExcludeMinR = 180.;
+    fExcludeMaxR = 250.;
+    break;
+  case 26: //p
```

Comment:
you missed case 25 here and thus the letter "p" will not correspond to case 26. Please fix

Reply:
sorry !

Reply:
No problem, it would just have crashed for you on the trains then. Thanks for fixing it.